lib/possible_bipartition.rb

```diff
@@ -1,4 +1,26 @@
 
 def possible_bipartition(dislikes)
-  raise NotImplementedError, "possible_bipartition isn't implemented yet"
+  return true if dislikes.empty?
+
+  i = 0 
+  while dislikes[i].empty?
+    i += 1
+  end
+
+  stack = []
+  visited = []
+  stack.push(i)
+
+  until stack.empty?
```

Comment:
You have a depth first search here, but how would this work for a disconnected graph?  I think may only be satisfying the tests based on the fact that you're starting with the first node that has a connection.